daemon/openrazer_daemon/misc/key_event_management.py

```diff
@@ -485,17 +486,18 @@ def key_action(self, event_time, key_id, key_press='press'):
                 elif self._recording_macro:
 
                     if self._current_macro_bind_key is None:
-                        # Restrict macro bind keys to M1-M5
+                        # Restrict macro bind keys to M1-M5                     comment out for KBDs without M1 ... M5 such as Ornata Chroma
                         if key_name not in ('M1', 'M2', 'M3', 'M4', 'M5'):
                             self._logger.warning("Macros are only for M1-M5 for now.")
                             self._recording_macro = False
                             self._parent.setMacroMode(False)
                         else:
                             self._current_macro_bind_key = key_name
                             self._parent.setMacroEffect(0x00)
-                    # Don't want no recursion, cancel macro, don't let one call macro in a macro
+                    # Don't want no recursion, cancel macro, don't let one call macro in a macro and be max. cryptic about it to confuse the user even more
```

Comment:
Please keep the text in a pull request (and also in issues) in a nice and respectful tone.